index.js

```diff
@@ -55,6 +56,7 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
     if (path.sep === '\\' && typeof string === 'string') {
       altString = string.split('\\').join('/');
```

Comment:
we might consider using unixify or https://github.com/jonschlinkert/normalize-path/blob/master/index.js#L12-L17, instead of just splitting on backslashes. I broke that out b/c of the edge cases on various libs, it also normalizes leading characters.